projects/ccf-rui/src/app/shared/components/organ-selector/organ-selector.component.spec.ts

```diff
@@ -74,4 +63,10 @@ describe('OrganSelectorComponent', () => {
     expect(instance.isSelected(testOrgan2)).toBeFalse();
   });
 
+  it('should set the icon class to disabled if disabled is true', async () => {
+    const { find } = await shallow.render();
+    const disabled = find('.carousel-item')[6].nativeElement as HTMLElement;
```

Comment:
You can't assume that the list from the component is going to stay the same (it'll eventually be data/ontology driven). You should use a fixed list like you had before here.